electron/app/utils/initUtils/index.ts

```diff
@@ -54,10 +58,43 @@ export async function coreInit(): Promise<sqlite3.Database> {
   if (possibleAddressBookDB) {
     log.info(`Contacts found: ${possibleAddressBookDB}`);
     await createContactTable(possibleAddressBookDB);
-    // await addContactColumn(lorDB);
+    try {
+      // Typescript thinks db.filename does not exist, but it does.
+      // @ts-ignore
+      const q = `ATTACH '${possibleAddressBookDB.filename}' AS ${addressBookDBAliasName}`;
+      await sqlite3Wrapper.runP(lorDB, q);
+      log.info(`ATTACH success`, q);
+    } catch (err) {
+      log.error('ERROR ATTACH errored', err);
+    }
+    try {
+      await addContactNameColumn(lorDB);
+      log.info(`ContactName column added successully`);
+    } catch (err) {
+      log.warn('WARN add ContactName column already exists', err);
+    }
+    try {
+      await setContactNameColumn(lorDB);
+      log.info(`ContactName Column set successully`);
+    } catch (err) {
+      log.error('ERROR setContactNameColumn error', err);
+    }
+    closeDB(possibleAddressBookDB); // after setContactNameColumn, we have no use for this db
   } else {
     log.info('No contacts found.');
   }
-  await createAllChatTables(lorDB); // TODO: all queries to use a coalesce
+  /*
+   * NOTE: Whether or not the addressBook was found, we
+   * can use a COALESECE(contact_name, handle.id)
+   * to return the name or the phone
+   */
+  await createAllChatTables(lorDB);
+  // TODO: remove this. Leaving this in here for now to ensure this works for other devs.
+  const result = await sqlite3Wrapper.allP(
+    lorDB,
+    'SELECT coalesce(contact_name, id) FROM handle'
+  );
+  console.log(result);
```

Comment:
This prints:

<img width="469" alt="Screen Shot 2020-12-27 at 12 31 54 PM" src="https://user-images.githubusercontent.com/29822597/103180347-d5b2a800-4849-11eb-81be-8d299b87192f.png">